lib/veri_frame.mli

```diff
@@ -0,0 +1,15 @@
+
+open Bap.Std
+open Bap_traces.Std
+open Regular.Std
+
+type t [@@deriving bin_io, sexp]
+type set = Value.Set.t
+
+val create: name:string -> trace:set -> bil:set -> t
+
+val name: t -> string
+val trace: t -> set
```

Comment:
`trace` and `bil` are bad names for a set of values got from a trace and from an interpretation. Especially in the public interface.